grails-app/services/au/org/ala/profile/hub/ProfileService.groovy

```diff
@@ -330,6 +334,32 @@ class ProfileService {
         resp
     }
 
+    def getBioStatus(String opusId, String profileId) {
```

Comment:
The profileId can be either the Scientific name OR the profile's UUID. You need to use the scientific name, so call profileService.getProfile() then explicitly reference the scientificName attribute. Note: getProfile actually returns both the opus and the profile, so you won't need to call getOpus() again.